distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestFederation.java

```diff
@@ -0,0 +1,219 @@
+                wrappedBundle(mavenBundle().groupId("org.jgrapht").artifactId("jgrapht-core")
+                        .versionAsInProject()),
+                keepRuntimeFolder());
+    }
+
+    @BeforeExam
+    public void beforeAllianceTest() throws Exception {
+        try {
+            basePort = getBasePort();
+            setLogLevels();
+
+            getServiceManager().waitForRequiredApps(REQUIRED_APPS);
+            getServiceManager().waitForAllBundles();
+            getCatalogBundle().waitForCatalogProvider();
+
+            configureSecurityStsClient();
+
+            startMockResources();
+            configureNsiliSource();
+        } catch (Exception e) {
+            LOGGER.error("Failed in @BeforeExam: ", e);
+            fail("Failed in @BeforeExam: " + e.getMessage());
+        }
+    }
+
+    @Test
+    public void testNsiliSourceAvailable() throws Exception {
+        // Determine if the Nsili Source has been configured in Alliance and available
+        // @formatter:off
+        given().auth().basic("admin", "admin").when().get(ADMIN_ALL_SOURCES_PATH.getUrl()).then()
+                .log().all().assertThat().body(containsString("\"fpid\":\"NSILI_Federated_Source\""));
+        // @formatter:on
+    }
+
+    @Test
+    public void testNsiliSourceOpenSearchGetAll() throws Exception {
+        // Simple search query to assert # records returned from mock source
+        ValidatableResponse response = executeOpenSearch("xml", "q=*", "src=" + NSILI_SOURCE_ID, "count=100");
+        response.body("metacards.metacard.size()", equalTo(11));
+    }
+
+    @Test
+    public void testNsiliSourceOpenSearchLocation() throws Exception {
+        // Perform query with location filtering
+        ValidatableResponse response = executeOpenSearch("xml", "q=*", "lat=-53.0", "lon=-111.0",
+                "radius=50", "src=" + NSILI_SOURCE_ID, "count=100");
+        response.log().all().body("metacards.metacard.size()", equalTo(11));
+    }
+
+    @AfterExam
+    public void afterAllianceTest() throws Exception {
+        if (mockServerThread != null) {
+            mockServerThread.interrupt();
+        }
+        mockServerThread = null;
+    }
+
+    private void startMockResources() throws Exception {
+        DynamicPort webPort = new DynamicPort("org.codice.alliance.corba_web_port", 6);
+        DynamicPort corbaPort = new DynamicPort("org.codice.alliance.corba_port", 7);
+
+        MockNsiliRunnable mockServer = new MockNsiliRunnable(Integer.parseInt(webPort.getPort()),
+                Integer.parseInt(corbaPort.getPort()));
+        mockServerThread = new Thread(mockServer, "mockServer");
+        mockServerThread.start();
+    }
+
+    private void configureNsiliSource() throws IOException {
+        NsiliSourceProperties sourceProperties = new NsiliSourceProperties(NSILI_SOURCE_ID,
+                "file://" + System.getProperty("user.dir") + "/../../ior.txt");
```

Comment:
Resolved. Refactored to remove use of IOR File, and serve IOR content via MockWebService. Relative paths are OBE.